src/components/global/selectableIconSet/SelectableIconSet.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { selectableIconMap } from '../../../utils/icon';
+
+interface SelectableIconSetProps {
+  setIcon: (iconKey: number) => void;
+}
+
+const SelectableIconSet: React.FC<SelectableIconSetProps> = ({ setIcon }) => {
+  const onClickIcon = (iconKey: string) => {
+    const iconNumber = parseInt(iconKey);
+    setIcon(iconNumber);
+  };
+  return (
+    <div
+      className='w-373 h-124 p-16 grid grid-cols-7 gap-y-12 gap-x-10 justify-items-center bg-white rounded-lg 
+    shadow-[0px_3px_4px_0px_rgba(24,24,24,0.06)] shadow-[3px_0px_4px_0px_rgba(24,24,24,0.06)] outline outline-1 outline-offset-[-1px] outline-primary-100'
+      onClick={(e) => e.stopPropagation()}>
+      {Object.entries(selectableIconMap).map(([key, Icon]) => (
+        <div
+          key={key}
+          className='w-40 h-40 flex items-center justify-center bg-primary-50 rounded outline outline-1 outline-offset-[-1px] outline-primary-100'
```

Comment:
이 부분 css 코드 중복됩니다 

```suggestion
    shadow-[3px_0px_4px_0px_rgba(24,24,24,0.06)] outline outline-offset-[-1px] outline-primary-100'
      onClick={(e) => e.stopPropagation()}>
      {Object.entries(selectableIconMap).map(([key, Icon]) => (
        <div
          key={key}
          className='w-40 h-40 flex items-center justify-center bg-primary-50 rounded outline outline-offset-[-1px] outline-primary-100'
```

그리고 지금은 아이콘 선택시 팝업이 닫히지 않는데 17번 줄의 `e.stopPropagation()` 때문인 것 같아요!
onClick 지우니까 아이콘을 선택하거나 모달 밖을 선택시에, 모달이 잘 닫아지더라구요.

Reply:
우선 css 코드 중복에 대해 말씀드리자면
중복된 값은 아니에요!
0px 3px 4px 0px는 아래로 그림자가 생기고,
3px 0px 4px 0p는 오른쪽으로 그림자가 생겨요
그래서 다른 값이 맞는데, 확인해보니 `shadow-[0px_3px_4px_0px_rgba(24,24,24,0.06)]` 적용이 안되더라고요?

이번에 알게된 사실인데 **tailwind는 shadow 속성을 하나만 인식해요**
현 속성값을 보니 그림자의 방향만 다르고 색상은 같아요.
그래서 하나의 속성값으로 합쳐서 문제 해결했습니다.
`shadow-[3px_3px_4px_0px_rgba(24,24,24,0.06)]` 
감사해요! 덕분에 아래 방향 그림자가 적용 안된다는 것을 알았어요

아이콘 선택시 팝업이 안 닫히는 것은 의도된거였어요.
화면 설계서에 해당 부분이 명시되있지 않아서, 어떻게 할까 고민하다가,
사용자가 원하는 아이콘을 계속 선택해보면서 가장 적합한 것을 정하지 않을까? 라는 생각에 이렇게 만들었습니다.
해당 부분은 기획자님께 한번 물어볼게요!

Reply:
넵 좋아요!! 그림자 값이 2개였던 사실은 제가 몰랐네요..! 저도 덕분에 tailwindcss가 같은 속성은 하나만 인식한다는 사실 알고갑니다 :)